### PR TITLE
Add some YAML

### PR DESCRIPTION
**Pull request by FedericoRondoni**
CDS7_windu_ERA5-SingleLevel_100m_GLOBAL CDS8_windv_ERA5-SingleLevel_100m_GLOBAL Modis_SNPP_2023 SSLC1_Seasonal_S1_AMP_hv_interferometric_coherence SSLC2_Seasonal_S1_COH12_hh_interferometric_coherence OHC300_Ocean_Heat_Content_upper_300m ENSST_by_GCOM-W-AMSR_JAXA SLSTR1_Sentinel-3-SLSTR-L2-LST SLSTR2_Sentinel-3-SLSTR-L2-WST NELST_ECOSTRESS_LST

### CODE REVIEW

**Review by aapopescu — COMMENTED** [review]
@FedericoRondoni  please see the comments

File: collections/CDS7_windu_ERA5-SingleLevel_100m_GLOBAL.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+Name: windu_ERA5-SingleLevel_100m_GLOBAL
+Title: Wind U Field at 100m
```
[the remark this answers is not in the document]
Comment (lubojr) [reply collections/CDS7_windu_ERA5-SingleLevel_100m_GLOBAL.yaml:2]: If the dataset is the same, you can reference the same YAML in both catalogs.

**Comment by lubojr** [comment]
@FedericoRondoni I've merged latest catalog generator software update to this branch to get some feedback on why the collections are failing.

One of the fixes done is that if the STAC API (Sentinel Hub) to get layer dates returns an empty list, the whole collection will not be faulty and only warning will be raised in the catalog generator.

- [ ] You can see this in here https://github.com/eurodatacube/eodash-catalog/actions/runs/10789477849/job/29922946107#step:9:30 - collection `c7fa984b-76f8-430a-8d57-2b8c4abe4e80` that it is empty.

- [ ] layers SENTINEL-3-SLSTR-L2-WST and SENTINEL-3-SLSTR-L2-LST are not configured on the SH instance. https://services.sentinel-hub.com/ogc/wms/0635c213-17a1-48ee-aef7-9d1731695a54?REQUEST=GetCapabilities
- [ ] layer `SST_BY_GCOM-W-AMSR_JAXA` is throwing following exception `<![CDATA[ Missing or empty evalscript. ]]>`
- [ ] layers `Seasonal_S1_COH12_hh_interferometric_coherence` and `Seasonal_S1_AMP_hv_interferometric_coherence` are not configured on the SH instance
- [x] Ocean heat content for the upper 300m is rendering fine as a WMS

**Comment by santilland** [comment]
@FedericoRondoni / @dmoglioni :
* change theme of Land Surface Temperature (Sentinel-3 SLSTR L2) to "extreme events"
* to recheck / update collection ids and layer ids
* provide evalscript to EOX so that we can generate legends

@AparicioSF :
* update markdowns for datasets within this branch
  - i.e. LST(2x), SST, active fires

@lubojr / @santilland :
* generate /add legends
* configure statistics api for SH datasets

**Comment by FedericoRondoni** [comment]
@lubojr @santilland the S1_AMP and S1_COH12 now should work. However, I see there is an issue with the SNPP yaml file. Can you check what it is and if it is something we have to change?
Thanks!

**Comment by lubojr** [comment]
@FedericoRondoni I have resolved the merge conflict about the SNPP file.

How do we decide [CDS7_windu_ERA5-SingleLevel_100m_GLOBAL.yaml](https://github.com/eurodatacube/eodash-catalog/pull/183/files/ad89013a01b336001eb9452d13c6d2403445e778#diff-ab39f114d0baa2ab269de76bc2a607919adda6b81f267e8f20d947237105e77e) and above comments from Anca? It is not necessary to add the same dataset YAML again when we already have the windu and windv variables from CDS in the RACE dashboard.

Are the datasets the same? If yes, it would just be necessary to reference the CDS3_wind_10m_u and CDS4_wind_10m_v and then delete possbly both CDS5_wind_100m_u_newDeli CDS6_wind_100m_v_newDeli and CDS7_windu_ERA5-SingleLevel_100m_GLOBAL  CDS8_windv_ERA5-SingleLevel_100m_GLOBAL?

**Comment by lubojr** [comment]
Confirmed on meeting that is good to merge once assets are finalized and merged. Latest friday midday.

**Pull request merged**